### ISSUE TITLE
Why plugin can clear a directory only BEFORE build?

### ISSUE THREAD

**Issue by akaSybe** [johnagan/clean-webpack-plugin#68]
What if I need to clear directory at the end of the build?

I need to build assets, create a zip file and remove everything in a directory except the zip file.
But looks like CleanWebpackPlugin everytime runs before all other plugins...

```
plugins: [
    new CleanWebpackPlugin(...),
    new CopyWebpackPlugin
    new StringReplacePlugin(),
    new ZipWebpackPlugin(...),
    new CleanWebpackPlugin(...)
]
```
Can you help me?

**Comment by clifgriffin** [comment]
I'm seeing this too.

**Comment by worenga** [comment]
Same here.

**Comment by jackypan1989** [comment]
remove old file after webpack build
https://github.com/jackypan1989/del-webpack-plugin

**Comment by chrisschrijver** [comment]
Just submitted a pull request to fix this:
https://github.com/johnagan/clean-webpack-plugin/pull/72

**Comment by johnagan** [comment]
Hopefully resolved in #72.

**Comment by chrisblossom** [comment]
For `v2` use `cleanAfterEveryBuildPatterns` to solve this issue. [Additional v2 Information can be found here](https://github.com/johnagan/clean-webpack-plugin/issues/106).
